### PR TITLE
added support for stylable

### PR DESCRIPTION
**Pull request by tomrav**
**Changes proposed:**
Add icon and configuration for stylable css files using the `.st.css` suffix.

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

### CODE REVIEW

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -476,6 +476,7 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    { icon: 'stylable', extensions: ['.st.css'], languages: [languages.stylable], light: true, format: FileFormat.svg },
```
Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: Please remove `light` attribute. This is used if there are different icons for dark and light themes.

Comment (tomrav) [reply src/icon-manifest/supportedExtensions.ts]: 👍

File: src/icon-manifest/supportedExtensions.ts [hunk 2]
```diff
@@ -476,6 +476,7 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    { icon: 'stylable', extensions: ['.st.css'], languages: [languages.stylable], format: FileFormat.svg },
```
Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: Also, remove the `.st.css` from the `extensions` attribute. Using `language id` ensures that whatever extensions are specified in the vscode `stylable` extension will be used. And in this case this [extension](https://marketplace.visualstudio.com/items?itemName=wix.stylable-intelligence) has it defined @  https://github.com/wix/stylable-intelligence/blob/master/packages/client/package.json#L33

Comment (tomrav) [reply src/icon-manifest/supportedExtensions.ts]: Just to make sure I understood, by using our language-intelligence extension, and allowing it to determine the language of the file - vscode-icons will mark files with the Stylable icon on any file determined to be a Stylable stylesheet (by said language-intelligence)?

Comment (JimiC) [reply src/icon-manifest/supportedExtensions.ts]: Yes. The icon will lit as soon as the extension get's installed on `vscode`.  And any file extension defined in the 3rd party extension will be automatically supported. That's the power of the `language id`.

**Comment by JimiC** [comment]
@tomrav Also I noticed that your `master` branch is way behind ours. Can you update it please?

**Comment by JimiC** [comment]
@tomrav I push changes to the icon to comply with our art guidelines (we haven't fully documented them yet so don't try to look them up).

**Comment by tomrav** [comment]
Thanks for the help! 👍

**Pull request merged**